src/server/json_family.cc

```diff
@@ -28,30 +29,19 @@ namespace dfly {
 using namespace std;
 using namespace jsoncons;
 
-using JsonExpression = jsonpath::jsonpath_expression<json>;
+using JsonExpression = jsonpath::jsonpath_expression<JsonType>;
 using OptBool = optional<bool>;
 using OptLong = optional<long>;
 using OptSizeT = optional<size_t>;
 using OptString = optional<string>;
```

Comment:
I know its not your code but those aliases are not really improving readability

Reply:
I will remove this, I didn't want to touch a code that I didn't wrote, since this already made a huge diff because of all these little changes

Reply:
I concur. can be part of another PR

Reply:
It's very common in the language that you alias specialized types of pods
for example the built-in aliases in atomic: atomic_bool, atomic_char and
etc.

Reply:
I prefer not to change this. like you've said, this is not "my" code and its not related to these changes.